docs/advanced/api.rst

```diff
@@ -14,12 +14,24 @@ Installation
 Fedora/CentOS
 -------------
 
-To allow remote access to the control agent REST API, and for agent connections,
+To enable Flocker control service.
+
+.. task:: enable_flocker_control
+   :prompt: [root@control-node]#
+
+The control service needs to accessible remotely.
+To configure firewalld to allow access to the control service REST API, and for agent connections,
```

Comment:
If you follow the link, it looks like most of the uses there are uncapitalized. So I uncapitalized the link as well, instead.